src/instrument/mongoose_instrument_exometer.erl

```diff
@@ -166,10 +170,17 @@ exometer_labels(#{pool_tag := PoolTag}) ->
 exometer_labels(#{}) ->
     [].
 
--spec get_host_type_prefix(mongoose_instrument:labels()) -> mongooseim:host_type_or_global().
+-spec get_host_type_prefix(atom() | binary() | mongoose_instrument:labels()) ->
+    mongooseim:host_type_or_global().
+get_host_type_prefix(HostType) when is_binary(HostType) ->
+    get_host_type_prefix(#{host_type => HostType});
+get_host_type_prefix(global) ->
+    global;
```

Comment:
I would not put this as a branch for `get_host_type_prefix`, I think this is confusing. Maybe matching on `global` in `get_metric_value` and calling `get_host_type_prefix` with a map would be better? Or maybe something like renaming the old `get_host_type_prefix` to `get_host_type_prefix_from_labels` and moving the logic to a new `get_host_type` prefix function?

Reply:
I agree, this part of the code might be a bit confusing. I'll try to refactor it. Thanks for the suggestions.